src/software/ai/hl/stp/play/enemy_ball_placement_play_test.py

```diff
@@ -11,82 +12,183 @@
-            gc_command=Command.Type.FORCE_START, team=Team.BLUE
-        )
-        # Pass in placement point here - not required for all play tests
-        simulated_test_runner.gamecontroller.send_ci_input(
-            gc_command=Command.Type.BALL_PLACEMENT,
-            team=Team.YELLOW,
-            final_ball_placement_point=ball_final_pos,
-        )
-
-        # Force play override here
-        blue_play = Play()
-        blue_play.name = PlayName.EnemyBallPlacementPlay
-
-        simulated_test_runner.blue_full_system_proto_unix_io.send_proto(Play, blue_play)
-
-        yellow_play = Play()
-        yellow_play.name = PlayName.BallPlacementPlay
-
-        simulated_test_runner.yellow_full_system_proto_unix_io.send_proto(
-            Play, yellow_play
-        )
-
-        # Create world state
-        simulated_test_runner.simulator_proto_unix_io.send_proto(
-            WorldState,
-            create_world_state(
-                yellow_robot_locations=yellow_bots,
-                blue_robot_locations=blue_bots,
-                ball_location=ball_initial_pos,
-                ball_velocity=tbots.Vector(0, 0),
+    yellow_play = Play()
+    yellow_play.name = PlayName.BallPlacementPlay
+
+    simulated_test_runner.yellow_full_system_proto_unix_io.send_proto(Play, yellow_play)
+
+    # Create world state
+    simulated_test_runner.simulator_proto_unix_io.send_proto(
+        WorldState,
+        create_world_state(
+            yellow_robot_locations=yellow_bots,
+            blue_robot_locations=blue_bots,
+            ball_location=ball_initial_pos,
+            ball_velocity=tbots.Vector(0, 0),
+        ),
+    )
+
+    # Always Validation
+
+    # Ball Placement Variables
+    ball_placement_segment = tbots.Segment(ball_initial_pos, ball_final_pos)
+
+    # Eventually Validation
+    eventually_validation_sequence_set = [
+        [
+            RobotEventuallyExitsRegion(
+                regions=[tbots.Polygon.fromSegment(ball_placement_segment, 0, 0.5)]
             ),
-        )
-
-    # TODO- #2783 Validation
-    # params just have to be a list of length 1 to ensure the test runs at least once
-    simulated_test_runner.run_test(
-        setup=setup,
-        params=[0],
-        inv_always_validation_sequence_set=[[]],
-        inv_eventually_validation_sequence_set=[[]],
-        ag_always_validation_sequence_set=[[]],
-        ag_eventually_validation_sequence_set=[[]],
-        test_timeout_s=7.5,
+            RobotEventuallyExitsRegion(regions=[tbots.Circle(ball_initial_pos, 0.5)]),
+            RobotEventuallyExitsRegion(regions=[tbots.Circle(ball_final_pos, 0.5)]),
+        ]
+    ]
+
+    # TODO: uncomment after fix AI behavior #2870
+
+    # simulated_test_runner.run_test(
+    #    eventually_validation_sequence_set=eventually_validation_sequence_set,
+    #    always_validation_sequence_set=always_validation_sequence_set,
+    #    test_timeout_s=7.5,
+    # )
```

Comment:
eventually_validation_sequence_set and always_validation_sequence_set got renamed in a previous PR. Can you update it according to the new arguments and parameters?